django/website/local_settings.py.dev

```diff
@@ -5,13 +5,13 @@ DEBUG = True
 DATABASES = {
```

Comment:
I didn't intend to commit these changes. They must have been included by mistake during merge